website/blog/2023-4-10-incrementals-pt2.md

```diff
@@ -0,0 +1,538 @@
+---
+slug: incremental-backups-pt2
+title: "Incremental Microsoft 365 backups in action"
+description: "In part 1 we discussed how there’s more than one way to run backups, and how full and incremental backups differ. With all the background information out of the way, it’s time to see how incremental backups actually come together in Corso. To do this, we’ll discuss things in the context of a running example."
+authors: amartinez
+tags: [corso, microsoft 365, backups]
+date: 2023-04-10
+image: ./images/incremental-scale.png
+---
+<!-- vale Vale.Spelling = NO -->
+
+![diagram of an incremental scale encoder By Lambtron - Own work, CC BY-SA 4.0, https://commons.wikimedia.org/w/index.php?curid=81494644](./images/incremental-scale.png)
+<!-- vale Vale.Spelling = YES -->
+
+In [part 1](https://corsobackup.io/blog/incremental-backups-pt1/) we discussed how there’s more than
```

Comment:
Please use relative links internally.

Reply:
Relative links are in, looking to see if this passes `make check` now